### PR TITLE
feat(community-members): break down visuals - INNO-356

### PR DESCRIPTION
**Pull request by kalinchernev**
Feel free to correct me if another separation of new components is considered better.

### CODE REVIEW

File: src/framework/atoms/text/README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+
+- Variation of a heading (`.heading--bold`)
+
+![Heading bold](./expected-heading--bold.jpg)
+
+- Variation of a list
+
+![List secondary](./expected-list--secondary.jpg)
+
+- Variation of image: circle image (Bootstrap's `.img-circle` can be re-used if available)
```
Comment (yhuard) [inline src/framework/atoms/text/README.md]: Indeed, should be moved into "image" (atom)

File: src/framework/molecules/meta/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Meta
```
Comment (yhuard) [inline src/framework/molecules/meta/README.md]: Does it really need to be a molecule? It's just text and links... 😢

Comment (kalinchernev) [inline src/framework/molecules/meta/README.md]: Moved to atoms. Maybe create a ticket to have a `Content` section for text, images, etc.?

File: src/framework/atoms/text/README.md (hunk 1 above)
Comment (kalinchernev) [inline src/framework/atoms/text/README.md]: This one moved as well.

File: src/framework/molecules/meta/README.md (hunk 2 above)
Comment (yhuard) [inline src/framework/molecules/meta/README.md]: text, images... they're all atoms 😉

Comment (kalinchernev) [inline src/framework/molecules/meta/README.md]: Yeah, sure - the idea was a bit different. Making the structure similar to ECL.

File: src/SUMMARY.md [hunk 3]
```diff
@@ -13,22 +13,26 @@
...
         *   [✓ Table](./framework/atoms/table/README.md)
-        *   [✓ Text](./framework/atoms/text/README.md)
+        *   [✎ Text](./framework/atoms/text/README.md)
     *   [Molecules](./framework/molecules/README.md)
         *   [✓ Breadcrumb](./framework/molecules/breadcrumb/README.md)
         *   [✓ Calendar](./framework/molecules/calendar/README.md)
         *   [✓ Community create button](./framework/molecules/community-create-button/README.md)
         *   [✓ Community navigation](./framework/molecules/community-navigation/README.md)
         *   [✓ FR Megamenu](./framework/molecules/fr-megamenu/README.md)
+        *   [✗ Meta](./framework/molecules/meta/README.md)
```
Comment (yhuard) [inline src/SUMMARY.md]: Not used anymore ;)

File: src/framework/atoms/text/README.md (hunk 1 above)
Comment (yhuard) [inline src/framework/atoms/text/README.md]: So you can remove those lines ;)

File: src/framework/molecules/user-info/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# User info
+
+{% hint style='danger' %}Planned{% endhint %}
+
+## Expected
+
+![Meta](./expected.jpg)
```
Comment (yhuard) [inline src/framework/molecules/user-info/README.md]: `[Meta]`?

File: src/framework/organisms/user-profile/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Twitter feed
```
Comment (yhuard) [inline src/framework/organisms/user-profile/README.md]: `Twitter feed`?

File: src/framework/molecules/user-info/README.md (hunk 4 above)
Comment (kalinchernev) [inline src/framework/molecules/user-info/README.md]: @yhuard, removed this i nthe same style as meta not being necessary any more. Just left the user-profile.

**Pull request merged**